src/transformers/data/processors/glue.py

```diff
@@ -81,26 +82,16 @@ def _tf_glue_convert_examples_to_features(
 
         def gen():
             for ex in features:
-                yield (
-                    {
-                        "input_ids": ex.input_ids,
-                        "attention_mask": ex.attention_mask,
-                        "token_type_ids": ex.token_type_ids,
-                    },
-                    ex.label,
-                )
+                d = {k: v for k, v in asdict(ex).items() if v is not None}
+                label = d.pop("label")
+                yield (d, label)
+
+        input_names = ["input_ids"] + tokenizer.model_input_names
```

Comment:
are you sure it's not already in there? (cc @LysandreJik)

Reply:
Yes, contains only `attention_mask` and `token_type_ids`. I think `input_ids` should be added. But I let @LysandreJik to confirm that.

Reply:
Hmm `input_ids` are always returned, so there's no need to specify it. The encode methods cannot not return `input_ids`.

Reply:
Do you prefer to update to something else or we keep it like this?

Reply:
looks good to me!